### DISCUSSION TITLE
Migrate may have effects on website

### DISCUSSION THREAD

**Discussion by ErikQQY** [facebook/docusaurus#5438] [General] [answered]
Hello guys, I want to migrate website based on Docusaurus v1 to Docusaurus v2, but while the original website already got a high ranking on google, I want to know if the migration may affect the ranking or other things, thanks a lot!!

**Comment by slorber** [comment, marked as answer]
Any migration can affect ranking, and you have to follow SEO migration guides carefully to make sure to not lose your ranking.

I'm not an SEO expert, but I believe if you just configure server-side redirects appropriately so that v1 URLs still exist in v2, you should be fine.

For i18n sites, the route pattern has slightly changed, from `/docs/<lang>/*` to `/<lang>/docs/*`, and you might want to add server redirects for that.

If you are using GitHub pages, I suggest to migrate to Netlify/Vercel first, because GitHub page does not provide server-side redirects features.

**Reply by ErikQQY** [reply]
Thanks for your response!!!!
